### PR TITLE
Blog: Blog post for scaling OpenTelemetry Collectors using Ansible

### PR DESCRIPTION
**Pull request by ishanjainn**
**Preview**: https://deploy-preview-4140--opentelemetry.netlify.app/blog/2024/scaling-opentelemetry-collectors/

### CODE REVIEW

**Review by chalin — COMMENTED** [review]
Other than the step numbering, as noted by @tiffany76, the indentation is wrong:

> image

Any content of a numbered list item should be indented by as many spaces as the number of characters before the first word of the item. (In your current format, that's 3 spaces, but if/when you fix the formatting, that might change.)

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:25]:
```suggestion
A [OpenTelemetry Collector role](https://github.com/grafana/grafana-ansible-collection/tree/main/roles/opentelemetry_collector) is provided through the [Grafana Ansible collection](https://docs.ansible.com/ansible/latest/collections/grafana/grafana/) as of the 1.1.0 release.
```

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:21]: I guess the end user also needs to have ansible installed?

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md]: I would recommend to replace these IPs with non-public ones, so something like 10.x.x.x, etc.

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md]:
```suggestion
{{% alert title="Note" %}}

If you are copying the above file, remove the comments (#).


{{% /alert %}}
```

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:85]:
```suggestion
```

Make this config independent of Grafana Cloud, we want to have this generally applicable.

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:105]:
```suggestion
```

Make this config independent of Grafana Cloud, we want to have this generally applicable.

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:152]: Replace this with jaeger, or use a debug exporter

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:158]: replace this with a debug exporter

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:165]: replace this with prometheus onprem

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:169]:
```suggestion
```

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:185]:
```suggestion
{{% alert title="Note" %}}

You'll need to adjust the configuration to match the specific telemetry you intend to collect and where you plan to forward it. This configuration snippet is a basic example designed for traces, logs, and metrics collected using OTLP and forwarded to Jaeger and Prometheus. 

{{% /alert %}}
```

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:204]:
```suggestion
```

see above, no vendor-specific details

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:206]: Make use of OSS Grafana for visualization or jaeger/prometheus standalone

**Review by svrnm — CHANGES_REQUESTED** [review]
Thanks, see my inline comments.

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md]: this still reads like a marketing blob, I have no issues with promoting OSS grafana here, but ideally this is practical, so how to setup grafana swiftly to see some of the data.

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [inline content/en/blog/2024/scaling-opentelemetry-collectors.md:6]: add a cSpell:ignore: section for words unknown to cspell, check other blog posts for examples

File: content/en/blog/2024/scaling-opentelemetry-collectors.md
Comment (svrnm) [reply content/en/blog/2024/scaling-opentelemetry-collectors.md]: yes, exactly.

**Comment by svrnm** [comment]
> Hey @svrnm The REFCACHE workflow tell me to run `npm run test-and-fix` but that doesnt seem to do anything for me

do not worry too much about that, we fix the checks at the end before merging the PR, there are some nuances with links that sometimes break the pipeline, and while updating the content you may break it again and again

> also for the `canonical_url` This can be added later right as I would be publishing a similar blog (for Grafana Cloud usage as OSS is covered here) but on Grafana's Blog

So, here is my semi-professional SEO opinion: the content that comes first is the "canonical url", so the grafana blog should point to this blog as canoncial. If the grafana blog has a policy against that, we need to discuss that.

**Comment by svrnm** [comment]
> > So, here is my semi-professional SEO opinion: the content that comes first is the "canonical url", so the grafana blog should point to this blog as canoncial. If the grafana blog has a policy against that, we need to discuss that.
> 
> Ahh, I don't think that is possible, Ill get that if possible too but would be nice to have it from here aswell, That is possible to update later or needs to be in this PR?

Do you think that it is not possible or do you know that it is not possible? Can you please check with the Grafana Blog Team first and then follow up with us?

Our current policy does not cover this use case, so we would need look into this (cc @open-telemetry/docs-approvers). 

Personally I still think it doesn't feel right to update an existing blog post to point to a blog post that came later and by that downgrading it from a SEO perspective: whoever takes the blog post first has the major work of copyediting and reviewing.

**Pull request closed without merging**
